cmd/tidb-dashboard/main.go

```diff
@@ -61,25 +62,40 @@ func NewCLIConfig() *DashboardCLIConfig {
 	}()
 
 	cfg := &DashboardCLIConfig{
-		CoreConfig: &config.Config{
-			Ctx:     ctx,
-			Version: utils.ReleaseVersion,
-		},
+		CoreConfig: config.NewConfig(ctx, utils.ReleaseVersion),
```

Comment:
Is this actually creating a default config?

Reply:
I want this effect, but it was not used later.